pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -521,6 +532,49 @@ func (l LibvirtWrapper) SetupLibvirt() (err error) {
 	return nil
 }
 
+func getLibvirtLogsFilters() (logFilters string, enableDebugLogs bool) {
```

Comment:
I would call this `getLibvirtLogFilters()` since everything else uses "log filters" instead of "logs filters". The former sounds more correct to me as well, but I'm not a native speaker :)

Reply:
Changed! thanks